ecommerce/courses/models.py

```diff
@@ -129,6 +129,13 @@ def enrollment_code_product(self):
 
     def get_course_seat_name(self, certificate_type, id_verification_required):
         """ Returns the name for a course seat. """
+        custom_seat_name = getattr(settings, 'CUSTOM_COURSE_SEAT_NAME', '')
+        if custom_seat_name:
+            return 'Seat in {} {} course'.format(
```

Comment:
Could you check if this string needs to be marked for extraction? @Jacatove

Reply:
It is not a translatable string. @Squirrel18